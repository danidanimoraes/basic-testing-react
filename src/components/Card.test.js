import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Card from './Card';
import "../../setupTests";

it('sould render card component with shallow', () => {
    expect(shallow(<Card />).length).toEqual(1)
})