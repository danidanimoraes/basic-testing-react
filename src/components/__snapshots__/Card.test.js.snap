// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should shallow match snapshot 1`] = `ShallowWrapper {}`;
